backend/app.py

```diff
@@ -36,8 +48,73 @@ def enterNewWord():
 
     return jsonify({'word': newWord}), 201
 
+
+# SOCKETS
+@socket.on('message')
```

Comment:
You are handling messages on embedded types.
You can create your own types, use these to define handlers.
You've started doing this on -> backed/app.py:111

See: https://flask-socketio.readthedocs.io/en/latest/#sending-messages